src/pages/studyView/StudyViewPage.tsx

```diff
@@ -108,7 +109,12 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
                 }
 
                 this.store.updateCurrentTab(props.routing.location.query.tab);
-                this.store.updateStoreFromURL(query);
+                const newStudyViewFilter:StudyViewURLQuery = _.pick(props.routing.location.query, ['id', 'studyId', 'cancer_study_id', 'filters', 'filterAttributeId', 'filterValues']);
```

Comment:
@alisman do we have a way to only get properties exist in the type `StudyViewURLQuery`?

Reply:
I think if you use destructuring notation you will get typechecking